pylint/checkers/base/basic_error_checker.py

```diff
@@ -467,8 +496,16 @@ def _check_inferred_class_is_abstract(
             return
 
         if metaclass.qname() in ABC_METACLASSES:
```

Comment:
Let's make L498 a `if not` and then return. There is no need for this indentation here.